src/semver/Version.d

```diff
@@ -53,6 +53,10 @@ struct Version
     string toString ( ) const pure
     {
         import std.format;
+
+        if (!this.valid())
+            return "malformed";
```

Comment:
Maybe add something more distinctive, like `<malformed>`? Or `<invalid>` sounds better to me, as I guess an uninitialized value is not really malformed.